6/root/code/.gitkeep

```diff
@@ -0,0 +1,20 @@
+FROM outrigger/servicebase
```

Comment:
I think this just needs to be an empty file.

Reply:
Once the flatten PR is merged I expect this will go away.